scripts/utility-wfrp4e.js

```diff
@@ -961,7 +961,144 @@ class WFRP_Utility
         canvas.draw();
       }
     }
+
+  static async PlayContextAudio(item, context)
```

Comment:
This should be gated by a conditional checking for an option being on. I know some ~~heathens~~ people will not want these sounds.